jdi-light-bootstrap/src/main/java/com/epam/jdi/light/ui/bootstrap/asserts/RangeAssert.java

```diff
@@ -0,0 +1,53 @@
+package com.epam.jdi.light.ui.bootstrap.asserts;
+
+import com.epam.jdi.light.asserts.generic.UIAssert;
+import com.epam.jdi.light.common.JDIAction;
+import com.epam.jdi.light.ui.bootstrap.elements.common.Range;
+import org.hamcrest.Matcher;
+
+import static com.epam.jdi.light.asserts.core.SoftAssert.jdiAssert;
+import static org.hamcrest.Matchers.is;
+
+public class RangeAssert extends UIAssert<RangeAssert, Range> {
+
+    @JDIAction("Assert that '{name}' thumbValue b {0}")
+    public RangeAssert thumbValue(Matcher<Double> thumbValue) {
+        jdiAssert(Double.parseDouble(element.getValue()), thumbValue);
+        return this;
+    }
+
+    public RangeAssert thumbValue(double thumbValue) {
+        return thumbValue(is(thumbValue));
+    }
+
+    @JDIAction("Assert that '{name}' min volume {0}")
+    public RangeAssert minValue(Matcher<Double> min) {
+        jdiAssert(Double.parseDouble(element.core().getAttribute("min")), min);
```

Comment:
It is better to use `element.min()` instead of `Double.parseDouble(element.core().getAttribute("min"))`

Reply:
fixed